rhizome/bin/storage-key-tool

```diff
@@ -0,0 +1,104 @@
+#!/bin/env ruby
```

Comment:
There's a lot of interesting application code in this file, perhaps it should go in lib and have some tests

Reply:
done